src/tox_travis.py

```diff
@@ -1,3 +1,6 @@
+"""Make it easy to work with Tox and Travis."""
+from __future__ import print_function
```

Comment:
I'm using the print function to print the warning message to stderr instead of stdout.